src/lib/components/colors/Output.svelte

```diff
@@ -17,10 +18,17 @@
     format,
     inGamut: false,
   });
+  function copyOutput() {
+    void navigator.clipboard.writeText(targetColorValue);
+  }
 </script>
 
 <ul data-group="output {type}">
   <li>
+    <button on:click={copyOutput} type="button" data-btn="icon">
+      <Icon name="clipboard" />
+      <span class="sr-only">Click to copy</span>
+    </button>
```

Comment:
@jamesnw is there any way to add an attribute or even a timed event that we can show a "copied" version of the icon just after copying something to the clipboard?

Reply:
Added a Copy Button component in c55d070 that switches to a different icon for 1 second.